hack/terraform-quickstart/main.tf

```diff
@@ -25,6 +25,17 @@ resource "aws_instance" "worker_node" {
   }
 }
 
+resource "aws_instance" "master_node" {
+  ami           = "${data.aws_ami.coreos_ami.image_id}"
+  instance_type = "m3.medium"
+  key_name      = "${var.ssh_key}"
+  count         = "${var.additional_masters}"
```

Comment:
ah I see. Can we add a comment / todo to this effect? (assuming you can have comments in tf files)